src/frontend/komponenter/Fagsak/Vedtak/utils.ts

```diff
@@ -8,10 +8,7 @@ interface PeriodeMedBeløp {
     beløp: number;
 }
 export const summerBeløpForPerioder = (periodeListe: PeriodeMedBeløp[]): number => {
```

Comment:
Denne påvirker også feiltutbetalt valuta, men vi ønsker samme endring der.

Reply:
Er tekster for feilutbetalt valuta også i orden eller sier vi kr/mnd i teksten der?

Reply:
Vi har også fjerner "/mnd" der 😄